lua/nvim-tree/view.lua

```diff
@@ -373,8 +392,10 @@ end
 
 function M.setup(opts)
   local options = opts.view or {}
+  M.View.adaptive_size = options.adaptive_size
   M.View.side = options.side
   M.View.width = options.width
+  M.View.initial_width = options.width
```

Comment:
`view.width = "20%"` is a problem.

Reply:
indeed ! Completely forgot about this one